json-endpoints/class.wpcom-json-api-comment-endpoint.php

```diff
@@ -192,6 +193,9 @@ function get_comment( $comment_id, $context ) {
 					),
 				);
 				break;
+			case 'can_moderate':
+				$response[$key] = (bool) current_user_can( 'edit_comment', $comment_id );
```

Comment:
Looks like you missed spaces inside the square brackets. Could you add them?

Reply:
Sure, updated in 865d89a7210bbc67de1822eade87d4a016d1cd06.